admin-dev/themes/new-theme/js/components/grid.js

```diff
@@ -63,6 +69,27 @@ export default class Grid {
     this.$grid.on('click', refreshListActionId, () => this._onRefreshClick());
     this.$grid.on('click', showSqlActionId, () => this._onShowSqlQueryClick());
     this.$grid.on('click', exportSqlManagerActionId, () => this._onExportSqlManagerClick());
+
+    $('.reset-search').on('click', (event) => {
+      resetSearch($(event.target).data('url'));
+    });
+  }
+
+  /**
+   * Handles the column sorting using Table component
+   *
+   * @private
+   */
+  _handleSortingGrid() {
+    const $sortableTable = this.$grid.find('table.table');
+    new TableSorting($sortableTable).attach();
+  }
+
+  /**
+   * If any, enable Date pickers component on date inputs.
+   */
+  _enableDatePickers() {
+      datePicker();
```

Comment:
Is it 2 or 4 spaces for indent?

Reply:
2 here imo, I've tried to respect the indentation already existing on this file, I'm not really aware of rules we respect on javascript files :/

Reply:
i'll fix